core/unit_test/cuda/TestCuda_ReducerViewSizeLimit.cpp

```diff
@@ -86,6 +86,12 @@ struct MDArrayReduceFunctor {
   KOKKOS_INLINE_FUNCTION void final(value_type) const {}
 };
 
+template <class PolicyType>
+struct FunctorFor {
+  KOKKOS_FUNCTION
+  void operator()(const typename PolicyType::member_type& /*team*/) const {}
```

Comment:
You could do
```C++
struct FunctorFor {
  template <class TeamHandle>
  KOKKOS_FUNCTION
  void operator()(TeamHandle const&) const {}
};
```
or
```C++
struct FunctorFor {
  KOKKOS_FUNCTION
  void operator()(Kokkos::TeamPolicy<TEST_EXECSPACE>::member_type const&) const {}
};
```
and then you don't need to spell a template argument for your functor below.

Reply:
👍 443f81e3fd56be47b7a804847a0462ea8c5cba8d